pkg/resource/aws/aws_route_ext.go

```diff
@@ -8,16 +8,22 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (r *AwsRoute) String() string {
-	var destination string
+func (r *AwsRoute) Attrs() map[string]string {
+	attrs := make(map[string]string)
+	routeId, err := CalculateRouteID(r.RouteTableId, r.DestinationCidrBlock, r.DestinationIpv6CidrBlock)
```

Comment:
Quite sur the route should have an idea here instead of computing a new one ?

Reply:
Ok so i delete this field of the map and console.go will handle the Id himself.